### ISSUE TITLE
Cannot do anything

### ISSUE THREAD

**Issue by shinyaMyn** [mathieudutour/git-sketch-plugin#205]
### How did it happen?
1.commit, push, branch, pull, generate pretty diff...everything cannot work.
2.Plugin doesn't work with my new laptop. I just installed homebrew, nodebrew, node, yarn.

### Error log

```
{
	"nativeException": {},
	"line": 195,
	"column": 37,
	"sourceURL": "/Users/[mypcname]/Library/Application Support/com.bohemiancoding.sketch3/Plugins/Git.sketchplugin/Contents/Sketch/Export.js"
}
```

**Comment by shinyaMyn** [comment]
plugin worked before I installed homebrew, nodebrew, node, yarn.

**Comment by mathieudutour** [comment]
fixed in 0.12
